include/lsst/ctrl/events/EventSystem.h

```diff
@@ -95,7 +106,16 @@ class EventSystem {
      */
     void createReceiver(std::string const& hostName, std::string const& topicName, int hostPort = EventBroker::DEFAULTHOSTPORT);
 
-    /** 
+    /**
+     * @brief create a EventDequeuer which will receive message
+     * @param hostName the location of the message broker to use
+     * @param queueName the queue to receive messages from
+     * @param hostPort the port where the broker can be reached
+     * @throws lsst::pex::exceptions::RuntimeError if topic is already registered
+     */
+    void createDequeuer(std::string const& hostName, std::string const& queueName, int hostPort = EventBroker::DEFAULTHOSTPORT);
```

Comment:
Please move this above the definition of `createReceiver`, in order to keep the various overloads of each function together.